Scenes/DetailScreen/Cells/ReviewsTableViewCell.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+class ReviewsTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var contentLabel: UILabel!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var ratingLabel: UILabel!
+    @IBOutlet weak var avatarImage: UIImageView!
```

Comment:
You have the function fillData to set the value of these properties, so they should be private or fileprivate.